components/tls/Makefile

```diff
@@ -7,20 +7,12 @@ test:
 .GOLANGCILINT_VERSION := v1.48.0
 .GOLANGCILINT_PATH := $(shell go env GOPATH)/bin/golangci-lint/$(.GOLANGCILINT_VERSION)
 
-.PHONY: .install-golangci-lint
-.ONESHELL: .install-golangci-lint
-.install-golangci-lint:
-	@installed=false
-	if [ -e "${.GOLANGCILINT_PATH}/golangci-lint" ]; then
-		installed=true
-	fi
-	if ! $${installed}; then
-		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
+${.GOLANGCILINT_PATH}/golangci-lint: 
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
```

Comment:
Are we always installing go lint every time now? If so how long does it usually take?

Reply:
We are not. It should only install if its not there.

Reply:
It is not clear to me how this is done given that we are removing the logic to check if the location of golint exist?

Reply:
`${.GOLANGCILINT_PATH}/golangci-lint:` should only run if needed.